osu.Game/Screens/Edit/Editor.cs

```diff
@@ -1098,8 +1098,17 @@ private void updateSampleDisabledState()
 
         private void seekControlPoint(int direction)
         {
+            // Gives margin to seek back after last control point
+            double seekMargin = 0;
+            
+            if (clock.IsRunning) 
+            {
+                IAdjustableClock adjustableClock = clock;
+                seekMargin = 450 * adjustableClock.Rate;
```

Comment:
I don't think rate needs to be accounted for? Especially if we increase this to 1 second.

Reply:
Probably fine to leave it, whatever works.